database/src/main/kotlin/no/nav/su/se/bakover/database/beregning/BeregningMapper.kt

```diff
@@ -56,14 +56,16 @@ internal data class PersistertMånedsberegning(
     private val sats: Sats,
     private val satsbeløp: Double,
     private val fradrag: List<PersistertFradrag>,
-    override val periode: Periode
+    override val periode: Periode,
+    private val fribeløpForEps: Double = 0.0,
```

Comment:
Hvorfor har vi en defaultverdi her? Skal vel ikke være behov for det siden vi patcher?